usr/src/uts/common/fs/zfs/txg.c

```diff
@@ -216,10 +216,11 @@ txg_sync_start(dsl_pool_t *dp)
 	mutex_exit(&tx->tx_sync_lock);
 }
 
+#define	TXG_THREAD_MAXNAMELEN	MAXCOMLEN + 1
 static void
-txg_thread_enter(tx_state_t *tx, callb_cpr_t *cpr)
+txg_thread_enter(tx_state_t *tx, callb_cpr_t *cpr, char *thrname)
 {
-	CALLB_CPR_INIT(cpr, &tx->tx_sync_lock, callb_generic_cpr, FTAG);
+	CALLB_CPR_INIT(cpr, &tx->tx_sync_lock, callb_generic_cpr, thrname);
```

Comment:
Perhaps in the future.  I'm mostly just trying to reduce long-standing deltas.  :)